sdk/loadtestservice/load-testing-rest/samples-dev/sample.ts

```diff
@@ -0,0 +1,63 @@
+import AzureLoadTesting, { AzureLoadTestingClient } from "@azure-rest/load-testing";
+import { DefaultAzureCredential } from "@azure/identity";
+import dotenv from "dotenv";
+import { createReadStream } from "fs";
+
+const readStream = createReadStream("./sample.jmx");
+dotenv.config({ path: "../../sample.env" });
+
+async function main() {
+  const Endpoint = process.env["LOADTESTSERVICE_ENDPOINT"] || "";
+  const DISPLAY_NAME = "some-load-test";
+  const SUBSCRIPTION_ID = process.env["SUBSCRIPTION_ID"] || "";
+  const Client: AzureLoadTestingClient = AzureLoadTesting(Endpoint, new DefaultAzureCredential());
+
+  await Client.path("/loadtests/{testId}", "abc").patch({
```

Comment:
Can test Id be anything? How do you expect your customers to get this Id? If you expect them to generate a GUID, I'd suggest you show that, so that they don't get confused about how to get an Id.

```ts
// Create a new guid for the testId
const testId = uuid.v4();
await Client.path("/loadtests/{testId}", testId).patch({
```